File: Python/programmers/etc/level1/핸드폰_번호_가리기.py
def solution(phone_number):
    return "*"*(len(phone_number)-4) + phone_number[-4:]


phone_number = "01033334444"
print(solution(phone_number)) # "*******4444"

phone_number = "027778888"
print(solution(phone_number)) # "*****8888"

File: Java/programmers/etc/level1/핸드폰번호가리기.java
package CodeTest.Java.programmers.etc.level1;

public class 핸드폰번호가리기 {
    public String solution(String phone_number) {
        String answer = "";
        String[] nums = phone_number.split("");
        int n = nums.length;
        
        for (int i=0; i<n-4; i++) {
            answer += "*";
        }
        
        for (int i=4; i>0; i--) {
            answer += nums[n-i];
        }
        
        return answer;
    }
}
